UI/window-basic-settings.cpp

```diff
@@ -462,6 +462,9 @@ OBSBasicSettings::OBSBasicSettings(QWidget *parent)
 	HookWidget(ui->disableFocusHotkeys,  CHECK_CHANGED,  ADV_CHANGED);
 	HookWidget(ui->autoRemux,            CHECK_CHANGED,  ADV_CHANGED);
 
+	ui->simpleOutputVBitrate->setSingleStep(50);
+	ui->advOutFFVBitrate->setSingleStep(50);
```

Comment:
What if on top of this, we also added `->setSuffix(" Kbps");`?

(I only learned this was possible through your Audio Improvements PR)

Reply:
That is a great idea. I will probably add it as a commit in #1859